tools/otsdb-dump-restore/README.md

```diff
@@ -0,0 +1,62 @@
+# OpenKilda TSDB dump/restore tools
+
+This packet contains python package that provide `kilda-otsdb-dump-restore` CLI tool. The tool itself
+provides following commands:
+* `kilda-otsdb-dump` download data from an OpenTSDB endpoint.
+* `kilda-otsdb-restore` restore data to an OpenTSDB capable database.
+
+# Create docker image
+
+```bash
+make
+````
+
+## Usage
+The tool is inside a Docker container. To use the it, run the following command in your terminal:
+```
+docker run kilda-otsdb-dump-restore <command_to_use> 
+```
+Note: if you need to run it in localhost, you need to add the `--network="host"` docker flag.
+
+### Commands:
+### `kilda-otsdb-dump --help`
```

Comment:
This doesn't look good as H3. I suggest you use some text form for the header and the command as paragraph:
### Dump data from OpenTSDB
Use this command `kilda-otsdb-dump` ...

the same for the next header. To me, `Commands` header is redundant here